include/tvm/runtime/threading_backend.h

```diff
@@ -0,0 +1,31 @@
+/*!
+ *  Copyright (c) 2018 by Contributors
+ * \file threading_backend.h
+ * \brief Utilities for manipulating thread pool threads.
+ */
+#ifndef TVM_RUNTIME_THREADING_BACKEND_H_
+#define TVM_RUNTIME_THREADING_BACKEND_H_
+
+namespace tvm {
+namespace runtime {
+namespace threading {
+
+class ThreadGroup {
+  public:
+    virtual void Launch(std::vector<std::function<void()>> task_callbacks) =0;
+    // TODO: virtual void Release(size_t n) =0;
+    virtual void Join() =0;
+    virtual void SetAffinity() =0;
+    virtual size_t Size() =0;
```

Comment:
Can remove the virtual function if this is not necessary

Reply:
I think that it might be necessary: the trusted thread pool will only maintain a `size_t` counter of the number of threads in the untrusted pool.